src/components/Page/Navigation/OrganizationSelect.tsx

```diff
@@ -0,0 +1,33 @@
+import { Organization } from "@api/client/models/Organization";
+import Select from "@components/Select";
+import { getLocalizedLabel } from "@utils/content";
+import { useTranslations } from "next-intl";
+
+interface Props {
+	organizations: Organization[];
+	activeOrganization: Organization | null;
+	onSelectOrganization: (organization: Organization | null) => void;
+}
+
+export default function OrganizationSelect({ organizations, activeOrganization, onSelectOrganization }: Props) {
+	const t = useTranslations("Navigation");
+	const options = organizations.map((organization) => ({
+		value: organization.identifier,
```

Comment:
This extra key seems obsolete to me.

Reply:
I thought so, too, but it’s actually needed by our Select component (`react-select`) to figure out which option is currently selected (similar to a regular `<select>` component).